news/_posts/2022-12-03-playtest-20221203.md

```diff
@@ -0,0 +1,88 @@
+    <source src="{{ '/images/news/20221203-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221203-td.png' | relative_url }}" width="700" />
+  </video>
+</figure>
+
+We have added transient text notifications matching speech notifications. Now you can play without sound!
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221203-transient.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221203-transient.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221203-transient.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Nuke incoming!</figcaption>
+</figure>
+
+The fairly recent Dune film has brought many players to Dune 2000 and a decently sized multiplayer community was born. With the feedback from the community Dune 2000 has received a number of balance and polish changes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221203-dune.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221203-dune.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221203-dune.png' | relative_url }}" width="700" />
+  </video>
+</figure>
+
+Tiberian Sun has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221203-slope.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221203-slope.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221203-slope.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
+</figure>
+
+<br>
+
+Some of the more visible new features and improvements include:
+ * Measures against sound / notification overlap. No more u-u-u-u-unit lost
+ * Measures against spamming the multiplayer lobby and in-game chat
+ * The ability to mute players
+ * Support for disabling the chat in replays
+ * New Red Alert missions
+ * An ingame encyclopedia in Dune 2000
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * A change to the way speed is calculated for ground units
+ * A new map format which includes the map preview (minimap) in the map versioning (map hash)
+ * Loaders for more audio formats (e.g., mp3 and ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and changes to support the .NET 6 runtime
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys became rebindable
+ * Substantial progress towards localisation
+
+All in all, 615 pull requests were merged since last release and 552 issues were closed bringing the total number of open tickets down by 141.
```

Comment:
Do we need to update this too?

Reply:
I had updated not too long ago. It's about accurate and I can't really make it more accurate. Github tools aren't flexible enough